### ISSUE TITLE
Release planner - Breaking change suppression request is not showing any status

### ISSUE THREAD

**Issue by praveenkuttappan** [Azure/azure-sdk-tools#5961]
If a PR has any breaking change then release planner showing breaking change failure. Release planner has a feature to request breaking change failure suppression and clicking this button adds a comment on specs PR to request suppression. 

1. Release planner app is not showing any status once suppression request is submitted and this allows user to submit suppression request again and this creates a new duplicate comment on PR. 

E.g.  
https://github.com/test-repo-billy/azure-rest-api-specs/pull/2920#issuecomment-1505616567
https://github.com/test-repo-billy/azure-rest-api-specs/pull/2920#issuecomment-1505622869

2. Added a label to mark the breaking change as approved. I think we should show some information if breaking change is approved to avoid confusion.

![image](https://user-images.githubusercontent.com/55455725/231530666-3f97399a-2498-4469-ac2b-d9b33914ea75.png)

**Comment by maririos** [comment]
From @ccbarragan : "Do service teams know what 'request suppression' means? Will they know what will happen?"
We need to make sure to cover this in the new UI and help our users

**Comment by maririos** [comment]
Is there any documentation we can link to here?

**Comment by ladonnaq** [comment]
I was blocked by the release planner not recognizing that the label Approved-BreakingChanges was not recognized, which causes API Readiness to fail for a PR that is merged and approved.  

Screenshot of error with request submission button active (but the breaking changes were approved) 
![Image](https://user-images.githubusercontent.com/67656762/235807132-91fa7e98-e83b-47ed-90e0-ac3cf6d3531f.png)

Screenshot of the PR. The label Approved-BreakingChanges was added by Jeffrey R.   Also, the label Approved-OkToMerge was added which is the label that indicates that the API Spec has been approved and is ready to mergen when ready.  

![Image](https://user-images.githubusercontent.com/67656762/235807387-70ef8905-590d-4f8b-b839-817b6e08fa4a.png)

**Comment by ladonnaq** [comment]
Also, adding another as an example. This PR is the private repo and it has all of the labels that we would look at to manually validate a team's initial preview API spec is ready to be published to the public specs repo.   Screenshot of errors.  Link to the PR - https://github.com/Azure/azure-rest-api-specs-pr/pull/12045


![Image](https://user-images.githubusercontent.com/67656762/235810982-74e8a11e-190f-454b-b03b-465c6499d41a.png)

**Comment by maririos** [comment]
> Also, adding another as an example. This PR is the private repo and it has all of the labels that we would look at to manually validate a team's initial preview API spec is ready to be published to the public specs repo. Screenshot of errors. Link to the PR - [Azure/azure-rest-api-specs-pr#12045](https://github.com/Azure/azure-rest-api-specs-pr/pull/12045)
> 

I don't see what the problem in this one is. The view is reflecting what is happening in the PR. The TypeSpec Validation is fialing in PR

**Comment by maririos** [comment]
> Screenshot of the PR. The label Approved-BreakingChanges was added by Jeffrey R. Also, the label Approved-OkToMerge was added which is the label that indicates that the API Spec has been approved and is ready to mergen when ready.

This is indeed a bug. We will look into it

**Comment by maririos** [comment]
@praveenkuttappan did you cover this as part of https://github.com/Azure/azure-sdk-tools/issues/6066 ?

**Comment by praveenkuttappan** [comment]
@maririos Nope. Breaking change was not covered as part of 6066. I will file a separate issue for breaking change approval since current issue is created to  avoid suppression request multiple times.

**Comment by praveenkuttappan** [comment]
@ladonnaq I don't see any handling for TypeSpec validation suppression approval in current release planner. Is there a PR label for TypeSpec validation suppression?

**Comment by praveenkuttappan** [comment]
I think we can reduce the priority of this issue since we track blocking issue using https://github.com/Azure/azure-sdk-tools/issues/6140

This issue is to track a change to have an option to identify if suppression request is already sent or not.

**Comment by ladonnaq** [comment]
> @ladonnaq I don't see any handling for TypeSpec validation suppression approval in current release planner. Is there a PR label for TypeSpec validation suppression?

I believe this is gap - no label for TypeSpec validation suppression.  There should be one if our team is going to merge PRs that have TypeSpec validation errors.

**Comment by maririos** [comment]
> > @ladonnaq I don't see any handling for TypeSpec validation suppression approval in current release planner. Is there a PR label for TypeSpec validation suppression?
> 
> I believe this is gap - no label for TypeSpec validation suppression. There should be one if our team is going to merge PRs that have TypeSpec validation errors.

This means that we will need a label per CI job that is running and I am not confident this is the approach the team wants.
This specific topic is out of scope for us in the Release Planner and it is more about how the API Spec process goes and tooling is helping there. There is already a group of people working on this so we need to wait and see what the guidance should be

**Comment by maririos** [comment]
Conclusion was to remoe the `Request supression` form the apps until there is more understanding on the process for them. Addressed in `1.0.0-beta.8` and `1.0.0-beta.9`
